### PR TITLE
TE-1982 Footer hide container

### PR DESCRIPTION
**Pull request by Jonnygwi**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-1982)
[Related YouTrack issue](https://youtrack.lodgify.net/agiles/86-18/87-1073?issue=TE-1980)

### What **one** thing does this PR do?
- Hide the navigation container if there are no navigation items.
- Aligns the searchable dropdown in the footer

### Any other notes
image
Screenshot 2019-03-11 at 15 49 12

### CODE REVIEW

File: src/components/collections/Footer/component.js [hunk 1]
```diff
@@ -122,6 +124,7 @@ Component.defaultProps = {
   currencyValue: null,
   languageOptions: [],
   languageValue: null,
+  navigationItems: [],
```
Comment (marcdavi-es) [inline src/components/collections/Footer/component.js]:
```suggestion
  navigationItems: null,
```
I don't think we need the array for anything?

File: src/components/collections/Footer/component.js [hunk 2]
```diff
@@ -34,24 +34,26 @@ export const Component = ({
...
-              ({ text, subItems }, index) => (
-                <Menu.Item
-                  key={buildKeyFromStrings(text || subItems[0].text, index)}
-                >
-                  <Menu.Menu>
-                    {text && <Menu.Header>{toUpper(text)}</Menu.Header>}
-                    {subItems.map(getMenuItemMarkup)}
-                  </Menu.Menu>
-                </Menu.Item>
+    {size(navigationItems) > 0 && (
```
Comment (marcdavi-es) [inline src/components/collections/Footer/component.js]:
```suggestion
    {size(navigationItems) && (
```
is enough, no?

Comment (Jonnygwi) [reply src/components/collections/Footer/component.js]: The above isn't enough as it returns a `0` but `{!!size(navigationItems) && (` is.. What do you recon? In other places we've done ` > 0` like on the `socialMediaLinks`.

**Pull request merged**